changelogs/unreleased/4681-izturn-small.md

```diff
@@ -0,0 +1 @@
+add the field podAnnotations to crd/ContourConfiguration to enable adding more annotations to pod/envoy
```

Comment:
worth mentioning in this PR and others youve submitted what the behavior of the added configuration is, what the defaults are, and how user configuration interacts with the defaults

Reply:
done